src/clj/rems/api/forms.clj

```diff
@@ -88,15 +90,22 @@
       (ok (form/get-form-template form-id)))
 
     (POST "/create" []
-      :summary "Create form"
+      :summary "Create or edit form"
       :roles #{:owner}
-      :body [command CreateFormCommand]
-      :return CreateFormResponse
-      (ok (form/create-form! (getx-user-id) command)))
+      :body [command FormCommand]
+      :return FormResponse
+      (ok (form/create-or-edit-form! (getx-user-id) command)))
 
     (PUT "/update" []
```

Comment:
There is likely a need to make sure the different kinds of update endpoints are named well. This should probably be named `/update-form-state` or maybe even `/form/:form-id/update-state` as is common in REST APIs.

Reply:
I started doing this but found out that there are quite a lot of places that the change touches (considering that similar update-state logic with similarly-named API end points is also implemented for catalogue items, licenses, and resources), so opted to add a TODO comment at this point and to do it in a later patch.